src/client/utils/cookie.js

```diff
@@ -0,0 +1,12 @@
+const getCookie = name => {
```

Comment:
This one probably could be removed

Reply:
getCookie method is used inside redux action in the Main folder to identify is the user is set to a cookie.